shuup/core/basket/objects.py

```diff
@@ -525,7 +525,7 @@ def _cache_lines(self):  # noqa (C901)
                                 break
                         if orderable:
                             orderable_lines.append(line)
-                            orderable_counter[product.id] += quantity
+                            orderable_counter[product.id] = quantity
```

Comment:
what if the package contain the same product more than once?

Reply:
But this is the parent